tests/plugins/tabletools/columndeletion.js

```diff
@@ -0,0 +1,74 @@
+/* bender-tags: editor,unit */
```

Comment:
I'll prepare two sets of test with and without tableselection plugin to cover more cases.